src/Tools/dotnet-monitor/CollectionRules/Options/Triggers/AspNetResponseStatusOptions.cs

```diff
@@ -12,17 +14,34 @@ namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Triggers
     /// </summary>
     internal sealed class AspNetResponseStatusOptions
     {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_AspNetResponseStatusOptions_StatusCodes))]
         [Required]
-        public string StatusCodes { get; set; }
+        [MinLength(1)]
```

Comment:
Does it make sense to have path filters as a generic across all `AspNet*Options` triggers?

Maybe I'm thinking it would be worth it to have `AspNetPathFilters` which holds `IncludePaths` and `ExcludePaths` and that filter can be a trigger just by itself and can be included in each of these options classes.

Reply:
Nesting of options like that comes at making the options configuration more complex. For example, the request count trigger might end up looking like:

```json
{
  "RequestCount": 100,
  "PathFilters": {
    "Include": [..],
    "Exclude": [..]
  }
}

```

I think the reusability of the options is somewhat low and replicating these two properties across the options is not too difficult to do or maintain. I think the complexity of specifying this type of structure outweighs the reusability or compactness we get from an engineering perspective.

Another option is to have a base class that contains these properties, have all of the ASP.NET triggers derive from it, and then flatten the base schema into the deriving schema when generating the schema.

At the very least, would could add these onto a common interface that allows for some reusable utility code for matching the path to the filters.

Reply:
I implemented the interface for now. We can review the configuration surface area later and determine if we want to do this type of nesting of settings.